### DISCUSSION TITLE
Mobile App for upscayl?

### DISCUSSION THREAD

**Discussion by itz-Me-Pj** [upscayl/upscayl#923] [Ideas]
As mobile devices are getting more powerful why wouldn't we create a app that will make this product more feasable and reach more people.
Atleast we can give it a try and create a prototype and test the implemented models. I have experience in developing mobile apps.

**Comment by aaronliu0130** [comment]
This has been discussed before in... #404. Do you have any experience in compiling C++ binaries that require Vulkan for mobile?

**Reply by itz-Me-Pj** [reply]
Sadly no but I can look for it and try to figure it out. Actually I have experience with developing apps via react-native.
